packages/parse5/lib/tokenizer/index.ts

```diff
@@ -3050,20 +3052,6 @@ export class Tokenizer {
         }
     }
 
-    // Decimal character reference start state
-    //------------------------------------------------------------------
-    private _stateDecimalCharacterReferenceStart(cp: number): void {
```

Comment:
The _Decimal character reference start state_ is called in the _Numeric character reference state_ and immediately transitions to another state. Because of that, it will never be called in `_callState`, so let's inline it.

Reply:
See my comment above